engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/impl/history/HistoricProcessDefinitionRestServiceImpl.java

```diff
@@ -16,31 +16,26 @@
  */
 package org.camunda.bpm.engine.rest.impl.history;
 
-import org.camunda.bpm.engine.HistoryService;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.ws.rs.core.UriInfo;
+
 import org.camunda.bpm.engine.ProcessEngine;
 import org.camunda.bpm.engine.history.CleanableHistoricProcessInstanceReport;
 import org.camunda.bpm.engine.history.CleanableHistoricProcessInstanceReportResult;
 import org.camunda.bpm.engine.history.HistoricActivityStatistics;
 import org.camunda.bpm.engine.history.HistoricActivityStatisticsQuery;
 import org.camunda.bpm.engine.impl.HistoricActivityStatisticsQueryImpl;
-import org.camunda.bpm.engine.rest.dto.converter.DateConverter;
-import org.camunda.bpm.engine.rest.dto.history.HistoricActivityStatisticsDto;
-import org.camunda.bpm.engine.rest.dto.history.CleanableHistoricProcessInstanceReportResultDto;
 import org.camunda.bpm.engine.rest.dto.CountResultDto;
 import org.camunda.bpm.engine.rest.dto.history.CleanableHistoricProcessInstanceReportDto;
-import org.camunda.bpm.engine.rest.exception.InvalidRequestException;
+import org.camunda.bpm.engine.rest.dto.history.CleanableHistoricProcessInstanceReportResultDto;
+import org.camunda.bpm.engine.rest.dto.history.HistoricActivityStatisticsDto;
 import org.camunda.bpm.engine.rest.history.HistoricProcessDefinitionRestService;
 import org.camunda.bpm.engine.rest.impl.AbstractRestProcessEngineAware;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 
-import javax.ws.rs.core.MultivaluedMap;
-import javax.ws.rs.core.UriInfo;
-import javax.ws.rs.core.Response.Status;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-
 public class HistoricProcessDefinitionRestServiceImpl extends AbstractRestProcessEngineAware implements HistoricProcessDefinitionRestService {
 
   public static final String QUERY_PARAM_STARTED_AFTER = "startedAfter";
```

Comment:
Do we need to keep those `QUERY_PARAM_X` constants? I couldn't find any other caller except the `HistoricProcessDefinitionRestServiceImpl` itself